packages/collapse/src/index.js

```diff
@@ -5,6 +5,13 @@ export default function (Alpine) {
 
         if (! el._x_isShown) el.style.height = `${floor}px`
         if (! el._x_isShown) el.style.removeProperty('display')
+        // We use the hidden attribute for the benefit of Tailwind
+        // users as the .space utility will ignore [hidden] elements.
+        if (! el._x_isShown) el.hidden = true
+        // We use display:none as the hidden attribute has very low
+        // CSS specificity and could be accidentally overriden by a
+        // user.
+        if (! el._x_isShown) el.style.display = 'none'
```

Comment:
For these changes, it'd be more consistent with `x-show` if we only set `el.style.display = 'none'`.

I've added the `el.hidden` parts to try and be helpful to Tailwind users who might be using the [space between](https://tailwindcss.com/docs/space) utilities to add spacing between collapsed elements. Without using `el.hidden`, the CSS would add extra space around collapsed items, which would be annoying. With `el.hidden`, Tailwind will ignore collapsed items. I don't know for sure, but this might be a CSS pattern used in other codebases too?

Reply:
Instead of reading display none, you can probably remove line 7 (which removes the style set by x-show)

Reply:
Ah yes, that's smarter! I did wonder about what the `el.style.removeProperty()` call was for, but I forgot that `x-collapse` works on top of `x-show`. I've just pushed a change with your suggestion.